src/cli/run.rs

```diff
@@ -326,6 +326,14 @@ fn disambiguate_task_interactive<'p>(
         ..ColorfulTheme::default()
     };
 
+    // Ignore CTRL+C in the dialoguer prompt so that it shows the cursor again.
+    ctrlc::set_handler(move || {
```

Comment:
I am still in favor of using tokio if we can make it work :)

Reply:
Changed it back